modules/virt-customizing-storage-profile.adoc

```diff
@@ -109,23 +109,71 @@ status:
-
-[cols="1,1",options="header"]
-|===
-|Storage provider|Default behavior
-|rook-ceph.rbd.csi.ceph.com|Snapshot
-|openshift-storage.rbd.csi.ceph.com|Snapshot
-|csi-vxflexos.dellemc.com|CSI Clone
-|csi-isilon.dellemc.com|CSI Clone
-|csi-powermax.dellemc.com|CSI Clone
-|csi-powerstore.dellemc.com|CSI Clone
-|hspc.csi.hitachi.com|CSI Clone
-|csi.hpe.com|CSI Clone
-|spectrumscale.csi.ibm.com|CSI Clone
-|rook-ceph.rbd.csi.ceph.com|CSI Clone
-|openshift-storage.rbd.csi.ceph.com|CSI Clone
-|cephfs.csi.ceph.com|CSI Clone
-|openshift-storage.cephfs.csi.ceph.com|CSI Clone
-|===
\ No newline at end of file
+[id="virt-viewing-automatically-created-storage-profiles_{context}"]
+.Viewing automatically created storage profiles
+
+The system creates storage profiles for each storage class automatically. To view the list of storage profiles, run the following command:
+
+[source,terminal]
+----
+oc get storageprofile
+----
+
+To print the details of a particular storage profile, run the following command:
+
+[source,terminal]
+----
+oc describe storageprofile <name>
+----
+
+.Example storage profile details
+[source,yaml]
+----
+$ oc describe  storageprofile ocs-storagecluster-ceph-rbd-virtualization
+Name:         ocs-storagecluster-ceph-rbd-virtualization
+Namespace:
+Labels:       app=containerized-data-importer
+              app.kubernetes.io/component=storage
+              app.kubernetes.io/managed-by=cdi-controller
+              app.kubernetes.io/part-of=hyperconverged-cluster
+              app.kubernetes.io/version=4.17.2
+              cdi.kubevirt.io=
+Annotations:  <none>
+API Version:  cdi.kubevirt.io/v1beta1
+Kind:         StorageProfile
+Metadata:
+  Creation Timestamp:  2023-11-13T07:58:02Z
+  Generation:          2
+  Owner References:
+    API Version:           cdi.kubevirt.io/v1beta1
+    Block Owner Deletion:  true
+    Controller:            true
+    Kind:                  CDI
+    Name:                  cdi-kubevirt-hyperconverged
+    UID:                   2d6f169a-382c-4caf-b614-a640f2ef8abb
+  Resource Version:        4186799537
+  UID:                     14aef804-6688-4f2e-986b-0297fd3aaa68
+Spec:
+Status:
+  Claim Property Sets: <1>
+    accessModes:
+      ReadWriteMany
+    volumeMode:  Block
+    accessModes:
+      ReadWriteOnce
+    volumeMode:  Block
+    accessModes:
+      ReadWriteOnce
+    volumeMode:                   Filesystem
+  Clone Strategy:                  csi-clone <2>
+  Data Import Cron Source Format:  snapshot <3>
+  Provisioner:                     openshift-storage.rbd.csi.ceph.com
+  Snapshot Class:                  ocs-storagecluster-rbdplugin-snapclass
+  Storage Class:                   ocs-storagecluster-ceph-rbd-virtualization
+Events:                            <none>
+----
+<1> `Claim Property Sets` is an ordered list of `AccessMode`/`VolumeMode` pairs, which describe the PVC modes that will be used to provision VM disks.
+<2> The `Clone Strategy` line indicates the clone strategy to be used.
```

Comment:
Can we link to any documentation that describes the three different cloning strategies (snapshot, csi-clone, copy)?

Reply:
@aglitke But the only place in the documentation where the three cloning strategies are described is in this section ("Setting a default cloning strategy using a storage profile").

Reply:
@aglitke 
Hi, 
So what's the verdict - are you approving the PR?

Reply:
Yes.  I probably missed that due to the way the sections are linked (and not seeing the preview).